### ISSUE TITLE
Accordion: use aria-controls?

### ISSUE THREAD

**Issue by micmath** [bbc/gef-docs#53]
Our tutorial for Accordion does not use the `aria-controls` attribute, but this attribute _is_ included in the [W3C WAI-ARIA Authoring Guide example](https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html). Is it missing on purpose? If so can we add an explanation on why it isn't needed please?
